### ISSUE TITLE
Add globally accessible A/B testing opt-out

### ISSUE THREAD

**Issue by tmancey** [brave/brave-browser#36295]

**Comment by haddadi** [comment]
most A/B product/design tests might not have any user data (will a red lion logo be more effective on shields than a blue lion logo?) in which case, there is no user data involved.

**Comment by diracdeltas** [comment]
I noted this in a DM to Hamed, but A/B test data (griffin state) is sent in crash reports too not just P3A. I think we are all on the same page that it'd be good to have a global opt out for A/B testing, so my understanding is that we can do this 3 ways:

1. Tie it to P3A opt out, and P3A opt out also opts out of sending Griffin data in crash reports. This seems a bit non-intuitive to me.
2. Explain in settings that Brave does A/B testing and to truly opt out from A/B test data collection you have to disable both the P3A and crash report flags. 
3. Separate toggle for A/B testing.

**Comment by ShivanKaul** [comment]
FWIW I just don't think we need to treat A/B tests as special cases of data collection.

**Comment by tmancey** [comment]
Added `needs-discussion` label. Thanks

**Comment by diracdeltas** [comment]
Is this actually a duplicate of https://github.com/brave/brave-browser/issues/15711 ? It seems in there we had consensus to add a new toggle for opting out of "staged roll-outs, feature testing and user studies" which is even more comprehensive than this issue, which is just to opt out of user studies without actually removing the ability to get new griffin variations.

**Comment by ShivanKaul** [comment]
Yeah, that one is a full-on "no griffin for me" toggle. If we add a `Data Collection` subsection like is the plan in https://github.com/brave/brave-browser/issues/36363, it would be fine to add a Griffin opt-out toggle there, and that should cover this case as well.
